### ISSUE TITLE
Could I control the relative height of the objects within the image?

### ISSUE THREAD

**Issue by bw4sz** [prgumd/SeaDroneSim2#5]
Hello, thanks for this project. I am a machine learning developer working with offshore biodiversity surveys from airborne images. We have hundreds of potential classes, which are hard to annotate in the imagery, but that we can get other sources of images from. For example, specific bird species. Birds sit on top of the water, and potential fly above the water. Could I dive into the source code and try to influence the relative height of the objects to include these kinds of classes?

<img width="1787" alt="Screenshot 2024-11-24 at 5 25 48 PM" src="https://github.com/user-attachments/assets/0d3221f9-37ee-4413-9d3f-c9d733db30f9">

My plan to create 3d models is to generate them using photographs to create a 3D mesh. I welcome input on the feasibility on any ideas.

**Comment by luuckiest** [comment]
Yes, this is completely editable. Everything we can modify. We are currently working on detecting rhinos and elephants. If you are interested in collaborating, please email me: at xlin52@jhu.edu
